dax-prompts/dax-prompts-impl/src/main/java/com/duckduckgo/daxprompts/impl/ReactivateUsersToggles.kt

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2025 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.daxprompts.impl
+
+import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
+import com.duckduckgo.daxprompts.impl.ReactivateUsersToggles.Companion.BASE_EXPERIMENT_NAME
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.feature.toggles.api.ConversionWindow
+import com.duckduckgo.feature.toggles.api.FeatureTogglesInventory
+import com.duckduckgo.feature.toggles.api.MetricsPixel
+import com.duckduckgo.feature.toggles.api.MetricsPixelPlugin
+import com.duckduckgo.feature.toggles.api.Toggle
+import com.duckduckgo.feature.toggles.api.Toggle.State.CohortName
+import com.squareup.anvil.annotations.ContributesMultibinding
+import dagger.SingleInstanceIn
+import javax.inject.Inject
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    featureName = BASE_EXPERIMENT_NAME,
+)
+interface ReactivateUsersToggles {
+
+    @Toggle.DefaultValue(false)
+    fun self(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun reactivateUsersExperimentMay25(): Toggle
+
+    enum class Cohorts(override val cohortName: String) : CohortName {
+        CONTROL("control"), // no Dax prompts
+        VARIANT_1("variant1"), // show Duck Player Dax Prompt
+        VARIANT_2("variant2"), // show Browser Comparison Dax Prompt
+    }
+
+    companion object {
+        internal const val BASE_EXPERIMENT_NAME = "reactivateUsers"
+    }
+}
+
+@ContributesMultibinding(AppScope::class)
+@SingleInstanceIn(AppScope::class)
+class ReactivateUsersPixelsPlugin @Inject constructor(
+    private val inventory: FeatureTogglesInventory,
```

Comment:
You can inject `private val toggles: ReactivateUsersToggles` instead to avoid having to loop through the whole inventory.